go.mod

```diff
@@ -14,7 +14,7 @@ replace (
 require (
```

Comment:
I can't comment there but the replace directive for controller-tools is now `v0.8.0-2` and should be updated accordingly.

Reply:
oh yeah nice catch, updating the PR now 🚀 🙏